internal/configuration/validator/spnego.go

```diff
@@ -0,0 +1,28 @@
+package validator
+
+import (
+	"fmt"
+
+	"github.com/authelia/authelia/v4/internal/configuration/schema"
+)
+
+func ValidateSpnego(config *schema.Configuration, validator *schema.StructValidator) {
+	if config.SPNEGO.Disable == true {
+		return
+	}
+	if config.AuthenticationBackend.LDAP == nil {
```

Comment:
```suggestion
	if !config.SPNEGO.Enabled {
		return
	}
	
	if len(config.Keytab) == 0 {
	    validator.Push(fmt.Errorf("spnego: option 'keytab' must be provided"))
	}
	
	if config.AuthenticationBackend.LDAP == nil {
```

Example validation adjustments. Essentially if the keytab is not configured we'd fail configuration validation.